### ISSUE TITLE
make test failed with "STD_CELL_LIBRARY_CDL no such variable"

### ISSUE THREAD

**Issue by pwang7** [efabless/openlane#362]
**Describe the bug**
when running make test, the test broke, with the error "STD_CELL_LIBRARY_CDL no such variable"

**To Reproduce**
1. make test

**Expected behavior**
make test should pass

**Screenshots**

**Desktop (please complete the following information):**
 - OS: ubuntu 20.04
 - Openlane Version: master branch
 - open_pdks Version: default
 - skywater-pdk Version: default

**Additional context**
![Screen Capture_select-area_20210519120719](https://user-images.githubusercontent.com/7015700/118755313-3a26e000-b89b-11eb-9981-c1cbcf03585a.png)

**Comment by Luo2017** [comment]
Hi，there! 
I have encountered the same problem while I'm trying to execute the command `make test` just like u on Ubuntu 20.04 and v0.16 of it. And I found that this variable is defined in ./scripts/tcl_commands/cvc.tcl:

![v0 16](https://user-images.githubusercontent.com/73009693/119018227-76a72880-b9ce-11eb-975b-dad5c6a98ec6.png)


I doubt whether the definition of this variable really works😶. And in v0.15, I found a different expression like this without using the variable:


![v0 15](https://user-images.githubusercontent.com/73009693/119018751-02b95000-b9cf-11eb-8aa0-588bbeeeebb8.png)


So I modified this file to make it just look like v0.15(or simply replace it):


![modified_file](https://user-images.githubusercontent.com/73009693/119020685-1bc30080-b9d1-11eb-8cb5-fa5e6db1e6a0.png)


And it works.


![success](https://user-images.githubusercontent.com/73009693/119020884-56c53400-b9d1-11eb-8358-bdce33b51d1a.png)

Hope it is useful to this issue and the problem should be really solved as soon as possible.✨
